helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
+  verbs:
+    - get
+    - list
+    - patch
+    - update
```

Comment:
I do not think we need patch and update on `apps` as the operator should not be allowed to change it's own CRs, but only the status field below

Reply:
Problem is `finalizers` and `annotations`.

Reply:
we do not have any `finalizers` and `annotations` in form of resource version we have only in chart CRD, so I think it's ok to leave those out as long as we do not use that functionality.